src/sentry/search/events/builder/base.py

```diff
@@ -77,7 +77,6 @@
 class BaseQueryBuilder:
     requires_organization_condition: bool = False
     organization_column: str = "organization.id"
-    uuid_fields = {"id", "trace", "profile.id", "replay.id"}
```

Comment:
Don't we still need the `uuid_fields` on the base query builder?

Reply:
Technically no because the only thing that checks is in the discover query builder but I've moved all the logic back into the base query builder and put this back.